kasa/smartcamera/smartcamera.py

```diff
@@ -29,6 +30,35 @@ def _get_device_type_from_sysinfo(sysinfo: dict[str, Any]) -> DeviceType:
             return DeviceType.Hub
         return DeviceType.Camera
 
+    @staticmethod
+    def _get_device_model_info(
+        info: dict[str, Any], discovery_info: dict[str, Any] | None
+    ) -> _DeviceInfo:
+        """Get model information for a device."""
+        basic_info = info["getDeviceInfo"]["device_info"]["basic_info"]
+        short_name = basic_info["device_model"]
+        long_name = discovery_info["device_model"] if discovery_info else short_name
+        brand = "tapo"
```

Comment:
I think I'd avoid introducing separate variables and just pass these as kwargs to the constructor. At least for the simple cases.